src/main/java/edu/usf/cutr/grha/ProcessorMain.java

```diff
@@ -16,32 +16,39 @@
 package edu.usf.cutr.grha;
 
 import edu.usf.cutr.grha.io.ChicagoTncParser;
+import edu.usf.cutr.grha.io.CsvParser;
 import edu.usf.cutr.grha.io.GPSTestExtendedHeaderParser;
 import edu.usf.cutr.grha.io.GPSTestParser;
 import edu.usf.cutr.grha.utils.IOUtils;
 
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+
 public class ProcessorMain {
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws FileNotFoundException {
```

Comment:
Instead of throwing `FileNotFoundException`, you should catch it and print out a message to the user that the file they specified can't be found.